sklearn/metrics/_plot/tests/test_roc_curve_display.py

```diff
@@ -302,7 +304,7 @@ def test_plot_roc_curve_pos_label(pyplot, response_method, constructor_name):
     classifier = LogisticRegression()
     classifier.fit(X_train, y_train)
 
-    # sanity check to be sure the positive class is classes_[0] and that we
+    # sanity check to be sure the positive class is `classes_[0]` and that we
     # are betrayed by the class imbalance
```

Comment:
@glemaitre I don't quite follow this comment?

Reply:
But what does "are betrayed by the class imbalance" mean? How does class imbalance affect `pos_label` ?

Reply:
I've amended the note, but let me know if it makes sense!